### PR TITLE
Implement initial look of the Header

### LINKED ISSUE #3: Implement Nav
- [ ] Desktop version
- [ ] Mobile version

### PR DESCRIPTION
**Pull request by jose-correia**
This PR implements the initial look of the header component.

The next steps involve:
- Add search tool
- Finalizing responsiveness for small screens
- Adding dropdown menus

Fix #3 

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/drafts/josec/
- After: https://navbar-initial-look--moleculardevices--hlxsites.hlx.page/drafts/josec/

### CODE REVIEW

File: blocks/header/header.js [hunk 1]
```diff
@@ -1,89 +1,31 @@
...
-  if (isNavDrop && (e.code === 'Enter' || e.code === 'Space')) {
-    const dropExpanded = focused.getAttribute('aria-expanded') === 'true';
-    // eslint-disable-next-line no-use-before-define
-    toggleAllNavSections(focused.closest('.nav-sections'));
-    focused.setAttribute('aria-expanded', dropExpanded ? 'false' : 'true');
-  }
+function buildToolsSection() {
+  const companyLinks = document.createElement('div');
+  companyLinks.innerHTML = `<div class="company-links"><ul class="mdtopmenu">
+  <li><em><i class="fa-briefcase" aria-hidden="true">&nbsp;</i></em><a href="https://jobs.danaher.com/global/en/molecular-devices" class="ext" data-extlink="" target="_blank" rel="noopener noreferrer">Careers</a></li>
```
Comment (mhaack) [inline blocks/header/header.js:6]: Can't we have that in the document as well, instead of hard code?

File: blocks/header/header.js [hunk 2]
```diff
@@ -1,89 +1,31 @@
...
+  <li class="dropdown"><em><i class="fa-globe" aria-hidden="true">&nbsp;</i></em><a class="dropdown-toggle" aria-expanded="false" aria-haspopup="true" data-toggle="dropdown" role="button">Language &nbsp;</a></li>
```
Comment (mhaack) [inline blocks/header/header.js:7]: Same, can't we have the list of languages in the doc?

File: blocks/header/header.js [hunk 3]
```diff
@@ -1,89 +1,31 @@
...
- * @param {Boolean} expanded Whether the element should be expanded or collapsed
- */
-function toggleAllNavSections(sections, expanded = false) {
-  sections.querySelectorAll('.nav-sections > ul > li').forEach((section) => {
-    section.setAttribute('aria-expanded', expanded);
-  });
+function buildRequestQuote() {
+  const requestQuote = document.createElement('li');
+  requestQuote.innerHTML = `<li class="we-mega-menu-li header-rfq" data-level="0" data-element-type="we-mega-menu-li" 
+  description="" data-id="38d3dbfb-400b-4a1a-8c62-23dfc88ea15d" data-submenu="0" data-group="0" data-class="header-rfq">
```
Comment (mhaack) [inline blocks/header/header.js:15]: Remove unused data attributes.

File: blocks/header/header.js [hunk 4]
```diff
@@ -1,89 +1,31 @@
...
-  // enable menu collapse on escape keypress
-  if (!expanded || isDesktop.matches) {
-    // collapse menu on escape press
-    window.addEventListener('keydown', closeOnEscape);
-  } else {
-    window.removeEventListener('keydown', closeOnEscape);
-  }
-}
+// function buildSearchTool() {
+//   const searchTool = document.createElement('li');
```
Comment (mhaack) [inline blocks/header/header.js:21]: Is this for later? If not please remove commented code.

File: blocks/header/header.js [hunk 5]
```diff
@@ -93,54 +35,62 @@ export default async function decorate(block) {
...
+  const navPath = getMetadata('nav') || '/drafts/josec/nav12';
+  const resp = await fetch(`${navPath}.plain.html`, window.location.pathname.endsWith('/drafts/josec/nav12') ? { cache: 'reload' } : {});
+  if (!resp.ok) return;
+
+  const html = await resp.text();
+
+  const logo = document.createElement('div');
+  logo.setAttribute('id', 'header-logo');
+  const logoSrc = '/images/moldev-logo.webp';
+  logo.innerHTML = `<div class="logo1"><a href="${navPath}"><img src="${logoSrc}" alt="Moleculardevices"/></a></div>`;
```
Comment (mhaack) [inline blocks/header/header.js:47]: The logo should be in the doc as well if possible.

File: blocks/header/header.js [hunk 6]
```diff
@@ -93,54 +35,62 @@ export default async function decorate(block) {
...
-    toggleMenu(nav, navSections, isDesktop.matches);
-    isDesktop.addEventListener('change', () => toggleMenu(nav, navSections, isDesktop.matches));
-
-    decorateIcons(nav);
-    const navWrapper = document.createElement('div');
-    navWrapper.className = 'nav-wrapper';
-    navWrapper.append(nav);
-    block.append(navWrapper);
+  const navPath = getMetadata('nav') || '/drafts/josec/nav12';
+  const resp = await fetch(`${navPath}.plain.html`, window.location.pathname.endsWith('/drafts/josec/nav12') ? { cache: 'reload' } : {});
```
Comment (mhaack) [inline blocks/header/header.js:39]: This looks like test code

Comment (jose-correia) [reply blocks/header/header.js:39]: Yup, nice catch

**Review by mhaack — APPROVED** [review]

**Pull request merged**